### DISCUSSION TITLE
Temperature shoots up in long run

### DISCUSSION THREAD

**Discussion by mayankaditya** [deepmodeling/deepmd-kit#288] [Q&A]
Hello All,

I ran MD simulation from deepmd potential. It works pretty well up to a few picoseconds and thereafter the temperature shoots up and atoms are getting very closed( it seems like dissociate). 
Should I train the model further? Do I need to include more training data or increase the cutoff?
Please suggest.

Thanks,
mayank

**Comment by njzjz** [comment]
I suggest to increase `sel` of your model. See https://github.com/deepmodeling/deepmd-kit/issues/108#issuecomment-545052796. You can check whether it’s enough in all snapshots.
